### PR TITLE
Eregcsc 1396,1399,1400

### PR DESCRIPTION
**Pull request by thwalker6**
Resolves #
1396
1399
1400
**Description-**

Using the PD concept added as the base I modified to use a lot of the same functionality.

Next and previous buttons to navigate through parts, subparts, and sections.

Can drill down to Subpart or section views by clicking on them.  If there is no content then it will be not a hyperlink.  Added a tooltip that says you can click it.

Added a section view.

This also is implementing V3 for api calls to pull a flat list of sections as well as retrieving content for the parts
**This pull request changes...**

- expected change 1

**Steps to manually verify this change...**

1. steps to view and verify change
 Go to {{server}}/zoom/{title}/{part}

You can hit previous next to go through parts.
Click a subpart.  Will drill down to subpart.  Hover over the link.  It will allow you to see a tooltip.  Hit forward and next to nav subparts.  Does not work for if you get to a grouped subpart placeholder ie. Subparts D-F
Click a section.  Will drill down to section. Can go forward and back.
Find and orphan section.  Cant go to next one since it is an orphan and its not in a subpart.

### CODE REVIEW

File: solution/ui/prototype/src/components/PDPart/LeftColumn.vue [hunk 1]
```diff
@@ -28,7 +28,7 @@
                     }"
                     >Next</router-link
                 >
-                <span v-else>Next</span>
```
Comment (PhilR8) [inline solution/ui/prototype/src/components/PDPart/LeftColumn.vue:31]: indenting

File: solution/ui/prototype/src/components/node_types/Section.vue [hunk 2]
```diff
@@ -120,4 +116,20 @@ export default {
...
+    margin:0;
+}
+.section-title a{
+    text-decoration: none;
+}
+.v-tooltip__content {
+    box-shadow: rgba(0, 0, 0, 0.3) 0 2px 10px;
+}
+.tooltip-text{
+    font-size: 12px !important;
```
Comment (PhilR8) [inline solution/ui/prototype/src/components/node_types/Section.vue:129]: indent

File: solution/ui/prototype/src/components/node_types/Subpart.vue [hunk 3]
```diff
@@ -1,30 +1,26 @@
...
+                class="supplemental-content-count">
                 {{ numSupplementalContent }}
             </button>
-            {{ node.title }}
+
+            <router-link :to="{
+                name: 'PDpart-subPart',
+                params: { title: this.title, part: this.part, subPart: 'Subpart-'+this.node.label[0]}
+            }">
+                <v-tooltip top color="#EEFAFE" ><template v-slot:activator="{ on, attrs }"><span v-bind="attrs"
```
Comment (PhilR8) [inline solution/ui/prototype/src/components/node_types/Subpart.vue:13]: formatting

File: solution/ui/prototype/src/views/PDPart.vue [hunk 4]
```diff
@@ -123,57 +126,70 @@ export default {
...
+            title: this.$route.params["title"],
+            part: this.$route.params["part"],
+            subPart: this.$route.params["subPart"],
+            section: this.$route.params["section"],
             supList: [],
             structure: [],
             subPartList: [],
             partsList: [],
             sections: [],
+            sections2:[],
```
Comment (PhilR8) [inline solution/ui/prototype/src/views/PDPart.vue:138]: section2 is unused

File: solution/ui/prototype/src/views/PDPart.vue [hunk 5]
```diff
@@ -123,57 +126,70 @@ export default {
     },
     data() {
         return {
-            title: this.$route.params.title,
-            part: this.$route.params.part,
-            subPart: this.$route.params.subPart,
-            section: this.$route.params.section,
+            title: this.$route.params["title"],
```
Comment (PhilR8) [inline solution/ui/prototype/src/views/PDPart.vue:129]: why move away from dot notation here?

File: solution/ui/prototype/src/views/PDPart.vue (hunk 4 above)
Comment (thwalker6) [reply solution/ui/prototype/src/views/PDPart.vue:138]: removed

File: solution/ui/prototype/src/components/node_types/Subpart.vue (hunk 3 above)
Comment (thwalker6) [reply solution/ui/prototype/src/components/node_types/Subpart.vue]: changed

File: solution/ui/prototype/src/components/node_types/Section.vue (hunk 2 above)
Comment (thwalker6) [reply solution/ui/prototype/src/components/node_types/Section.vue:141]: changed

**Review by PhilR8 — APPROVED** [review]

**Pull request merged**